src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -78,16 +108,17 @@ def find_date_in_string(string: str) -> Optional[str]:
         - 30.4.2021 -> GREGORIAN:CE:2021-04-30:CE:2021-04-30
         - 5/11/2021 -> GREGORIAN:CE:2021-11-05:CE:2021-11-05
         - Jan 26, 1993 -> GREGORIAN:CE:1993-01-26:CE:1993-01-26
-        - February26,2051 -> GREGORIAN:CE:2051-02-26:CE:2051-02-26
```

Comment:
superfluous example (just orthographic variance, doesn't need to be listed here)